Backend-Challenges.md

```diff
@@ -125,10 +272,79 @@ Additionally, most explorers will display the addresses of both the sender and r
 
 You can think of Etherscan as the Google of Ethereum. Just as you would use Google to search the internet, you can use Etherscan to search the Ethereum blockchain.
 
-If you paste a contract address in the top right search bar, you will land on the smart contract page, and you will be able to see all kinds of data and even interact with the smart contract by connecting your wallet. So don't be fooled if your front enders create some "security" measures that would prevent people from minting NFTs on the front end. When there is a will, there is a way. Either by using their own contracts or by simply going to Etherscan.
+If you paste a contract address in the top right search bar, you will land on the smart contract page, and you will be able to see all kinds of data and even interact with the smart contract by connecting your wallet.
 
-Now, go ahead and click on the "Contract" tab. Below that, you will see three tabs called "Code," "Read Contract," and "Write Contract."
+Now, go ahead and click on the `Contract` tab. Below that, you will see three tabs called `Code,` `Read Contract,` and `Write Contract.`
 
-- Code => All the smart contract code. Here, you can also get an idea of how NFTs were made back in the day.
+- Code => All the smart contract code. Here, you can also get an idea of how NFTs were made back in the day if you look at old contracts such as Bored Ape Yacht Club.
 - Read Contract => Interact with all the read-only functions of the smart contract, for example, `tokenUri()`.
 - Write Contract => Interact with all the write functions of the smart contract (Requires wallet connection), for example, minting.
+
+### Deploying the contract
+
+Now that you know what Etherscan is, it is time to deploy and verify our smart contract. If a smart contract is not verified, you cannot click on the functions to interact with them, nor see them directly. An unverified smart contract [will look like this](https://etherscan.io/address/0x6c830acd0b1610f547f568abd9500012cffd5208#code).
+
+As per the [Foundry documentation](https://book.getfoundry.sh/forge/deploying#deploying), we can verify and deploy our contracts at the same time!
+
+Here is a format you can copy and paste into your terminal to deploy and verify the contract.
+
+`forge create --rpc-url <alchemy_api_key> --private-key <wallet_private_key> smart-contracts/src/Challenge_12.sol:Challenge12 --constructor-args "NFT Example" "NFTEX" 100 --etherscan-api-key <etherscan_api_key> --verify`
+
+Below are the steps to acquire the necessary API keys:
```

Comment:
I think good to mention (again) they should ensure to have Eth in their wallet now. Good to re-reference to the faucet or let us knwo